book/website/reproducible-research/ci/ci-github-actions.md

```diff
@@ -0,0 +1,93 @@
+(rr-ci-ghactions)=
+# Continuous Integration with GitHub Actions
+
+This section will walk you through the basic setup of continuous integration (CI) using **GitHub Actions (GHA)**. GHA is a task automation system fully integrated with GitHub. In other words, it is an API that orchestrates any workflow based on any event. Although there are many CI service providers, GHA makes it easier than ever to incorporate CI into your repositories. It provides a flexible way to automate nearly every aspect of your project workflow. Here are just a few examples of use cases of GitHub Actions:
```

Comment:
maybe link to othe main github actions documentation again from the first mention of github actions

Reply:
Does API defined elsewhere? if not maybe rephrase that sentence